frame/nomination-pools/src/tests.rs

```diff
@@ -1942,10 +1943,269 @@ mod claim_payout {
+
+				// cannot go to below 10:
+				assert_noop!(
+					Pools::unbond(Origin::signed(kicker), 20, 15),
+					Error::<T>::PartialUnbondNotAllowedPermissionlessly
+				);
+
+				// but they themselves can do an unbond
+				assert_ok!(Pools::unbond(Origin::signed(20), 20, 2));
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().active_points(), 18);
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().unbonding_points(), 2);
+
+				// can be kicked to 0.
+				assert_ok!(Pools::unbond(Origin::signed(kicker), 20, 18));
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().active_points(), 0);
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().unbonding_points(), 20);
+			})
+	}
+
+	#[test]
+	fn member_unbond_destroying() {
+		// depositor in pool, pool state destroying
+		//   - member cannot be permissionlessly unbonded to above limit
+		//   - member cannot be permissionlessly unbonded between within limit and 0
+		//   - member permissionlessly unbonded to 0
+		ExtBuilder::default()
+			.min_join_bond(10)
+			.add_members(vec![(20, 20)])
+			.build_and_execute(|| {
+				unsafe_set_state(1, PoolState::Destroying);
+				let random = 123;
+
+				// cannot be kicked to above the limit.
+				assert_noop!(
+					Pools::unbond(Origin::signed(random), 20, 5),
+					Error::<T>::PartialUnbondNotAllowedPermissionlessly
+				);
+
+				// cannot go to below 10:
+				assert_noop!(
+					Pools::unbond(Origin::signed(random), 20, 15),
+					Error::<T>::PartialUnbondNotAllowedPermissionlessly
+				);
+
+				// but they themselves can do an unbond
+				assert_ok!(Pools::unbond(Origin::signed(20), 20, 2));
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().active_points(), 18);
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().unbonding_points(), 2);
+
+				// but can go to 0
+				assert_ok!(Pools::unbond(Origin::signed(random), 20, 18));
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().active_points(), 0);
+				assert_eq!(PoolMembers::<Runtime>::get(20).unwrap().unbonding_points(), 20);
+			})
+	}
+
+	#[test]
+	fn depositor_unbond_open() {
+		// depositor in pool, pool state open
+		//   - depositor  unbonds to above limit
+		//   - depositor cannot unbond to below limit or 0
+		ExtBuilder::default().min_join_bond(10).build_and_execute(|| {
+			// give the depositor some extra funds.
+			assert_ok!(Pools::bond_extra(Origin::signed(10), BondExtra::FreeBalance(10)));
+			assert_eq!(PoolMembers::<T>::get(10).unwrap().points, 20);
+
+			// can unbond to above the limit.
+			assert_ok!(Pools::unbond(Origin::signed(10), 10, 5));
+			assert_eq!(PoolMembers::<Runtime>::get(10).unwrap().active_points(), 15);
+			assert_eq!(PoolMembers::<Runtime>::get(10).unwrap().unbonding_points(), 5);
+
+			// cannot go to below 10:
+			assert_noop!(Pools::unbond(Origin::signed(10), 10, 15), Error::<T>::MinimumBondNotMet);
+
+			// cannot go to 0 either.
+			assert_noop!(Pools::unbond(Origin::signed(10), 10, 20), Error::<T>::MinimumBondNotMet);
```

Comment:
```suggestion
			assert_noop!(Pools::unbond(Origin::signed(10), 10, 10), Error::<T>::MinimumBondNotMet);

			// cannot go to 0 either.
			assert_noop!(Pools::unbond(Origin::signed(10), 10, 15), Error::<T>::MinimumBondNotMet);
```
The `active_points` are `15`, so deducing 15 would already result in 0.